src/workspacer.Bar/Widgets/TitleWidget.cs

```diff
@@ -39,7 +39,7 @@ private IWindow GetWindow()
             var currentWorkspace = Context.WorkspaceContainer.GetWorkspaceForMonitor(Context.Monitor);
             return currentWorkspace.FocusedWindow ??
                    currentWorkspace.LastFocusedWindow ??
-                   currentWorkspace.Windows.FirstOrDefault(w => w.CanLayout);
+                   currentWorkspace.Windows.FirstOrDefault(w => w?.CanLayout ?? false);
```

Comment:
This fixed an issue where when debugging (with Visual Studio) I would open the workspacer debug window and trigger an exception.